### ISSUE TITLE
LRScheduler assumes that lower scores are always better

### ISSUE THREAD

**Issue by benjamin-work** [dnouri/skorch#324]
However, some scores are better the higher they are, e.g. accuracy. Ideally, `LRScheduler` gets a `lower_is_better` argument, similar to the scoring callbacks.

Possibly, it is enough to adjust this line:

https://github.com/dnouri/skorch/blob/0f7823cb6a27b5d7dfab56f06a6601bf2957594b/skorch/callbacks/lr_scheduler.py#L116

to include `score = score if lower_is_better else -score`. I'm not sure how pytorch deals with negative scores.

**Comment by benjamin-work** [comment]
Thanks. Maybe we can re-open this if other schedulers use the score.
